dev-requirements.txt

```diff
@@ -17,7 +17,8 @@ pylint<2.0;python_version<="2.7"
 gcp-devrel-py-tools
 
 # optional dependencies, only intended for use with Python 3.5+
-trio==0.3.0; python_version >= "3.5"
+trio==0.13.0; python_version >= "3.5"
+pytest-trio==0.5.2;python_version>="3.5"
```

Comment:
I think these should be 3.6, not 3.5?

Also, might take more sense to use `>=` restrictions? Really it's a matter of preference. `==` is less likely to break unexpectedly, but requires more attention to keep up to date.

Reply:
I fixed the 3.6 issue. I'm not sure I want to turn all dependencies in >= just like that. I had two issues with twisted installation that took me a lot of time to fix, and many dependencies are soon going to drop Python 2.7 while we still support it, so I'd like to reduce the number of moving parts right now. It would be nice to configure dependabot though.